Mbox-Provider.md

```diff
@@ -5,19 +5,26 @@ The mbox provider is a Jakarta Mail local store provider that manages files
 in [ Unix mbox format ](http://en.wikipedia.org/wiki/Mbox)
 (in particular, the *mboxcl2* variant).
 
-The mbox provider has only been tested on Solaris and OpenSolaris. It
-should be easy to make it work on Linux, but I haven't gotten around to
-that yet. I tried it once on Windows, long ago, but there's no file
+The mbox provider has only been tested on Solaris, OpenSolaris
+and distributions based on Debian.
+I tried it once on Windows, long ago, but there's no file
 locking support for use on Windows. On Solaris and OpenSolaris it
 depends on native file locking code.
 
-To build the mbox provider:
+To build the mbox provider for Solaris/OpenSolaris:
 
     % (cd mbox; mvn)
     % (cd mbox/native; mvn)
 
+To build the mbox provider for distributions based on Debian:
+
+    % (cd mbox; mvn)
+    % (cd mbox/native; mvn linux)
```

Comment:
```
[ERROR] Unknown lifecycle phase "linux"
```

Reply:
Thank you for pointing me that. It is with -P